cmd/dev/snapshots.cpp

```diff
@@ -382,6 +435,21 @@ void download(const bittorrent::BitTorrentSettings& settings) {
     SILK_INFO << "Bittorrent async download completed for magnet file: " << *settings.magnets_file_path;
 
     scheduler_thread.join();
+}
+
+void download(const DownloadSettings& settings) {
+    std::chrono::time_point start{std::chrono::steady_clock::now()};
+
+    bittorrent::TorrentInfoPtrList web_seed_torrents;
+    if (!settings.url_seed.empty()) {
+        web_seed_torrents = download_web_seed(settings);
+
+        if (settings.download_web_seed_torrents) {
+            download_bittorrent(settings, web_seed_torrents);
+        }
+    } else {
+        download_bittorrent(settings, web_seed_torrents);
```

Comment:
If you don't specify any web seed URL, it will download the target files specified in input file containing a list of magnet links. Your doubt is absolutely legit, the code is not clear enough: I'm going to change it and make it more readable.